Code/GraphMol/TautomerQuery/catch_tests.cpp

```diff
@@ -308,3 +308,10 @@ TEST_CASE("TEST_NOT_TAUTOMER") {
   REQUIRE(target);
   CHECK(tautomerQuery->isSubstructOf(*target));
 }
+
+TEST_CASE("TAUTOMERQUERY_COPY_CONSTRUCTOR") {
```

Comment:
I think it's helpful to be able to relate issues directly to the tests for them:
```suggestion
TEST_CASE("github #3821 TAUTOMERQUERY_COPY_CONSTRUCTOR") {
```